book/src/drivers.md

```diff
@@ -72,7 +72,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [OpenSDS](https://github.com/opensds/nbp/tree/master/csi) | `csi-opensdsplugin` | v1.0 | A Container Storage Interface (CSI) Driver for [OpenSDS]((https://www.opensds.io/)) | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot
 [Open-E](https://github.com/open-e/JovianDSS-KubernetesCSI) | `com.open-e.joviandss.csi` | v1.0 | A Container Storage Interface (CSI) Driver for Open-E JovianDSS Storage  | Persistent | Read/Write Single Pod | Yes | Snapshot, Cloning 
 [Portworx](https://github.com/libopenstorage/openstorage/tree/master/csi) | `pxd.openstorage.org` | v0.3, v1.1 | A Container Storage Interface (CSI) Driver for [Portworx](https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/csi/) | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion
-[Pure Storage CSI](https://github.com/purestorage/helm-charts)| `pure-csi` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for Pure Storage's [Pure Service Orchestrator](https://purestorage.com/containers) | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Cloning, Raw Block, Topology, Expansion
+[Pure Storage CSI](https://github.com/purestorage/pure-pso)| `pure-csi` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for Pure Storage's [Pure Service Orchestrator](https://purestorage.com/containers) | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Cloning, Raw Block, Topology, Resize
```

Comment:
Is "https://github.com/purestorage/pure-pso" the right link?  It gave me a "404" error.

Reply:
"Expansion" is used for all other drivers in this doc.  Can you keep this as "Expansion"?

Reply:
@xing-yang fixed